src/lib/rtc-to-net/rtc-to-net.ts

```diff
@@ -663,6 +670,19 @@ import ProxyConfig = require('./proxyconfig');
       return true;
     }
 
+    private calculateBandwidth_ = () : void => {
+      if (!this.stopBandwidthCalc_) {
+        //calculate bandwidth over 5 second interval based on total bytes sent and received by this session
+        //we want to calculate bandwidth over a certain time interval; doing it continuously would not help us stop a random
+        //peak in badnwidth usage if the overall average is still low.
+        var bitsTransferred_ = (this.currBytesSession_ - this.prevBytesSession_) * 8; //8 bits in a byte, bandwidth measured in metric multiple of bits/sec 
+        var bandwidthSession_  = bitsTransferred_ / 5; //5 seconds have elapsed since last time this method was called
+
+        log.debug('It has been 5 sec in session' + this.channelLabel() + '; bandwidth: ' + bandwidthSession_ + ' bits/sec');
+        this.prevBytesSession_ = this.currBytesSession_;
```

Comment:
Let's extract a class-level constant for the interval, e.g. `BANDWIDTH_MONITOR_INTERVAL`.